test/cli.test.js

```diff
@@ -244,13 +244,6 @@ it('throws on --why with disabled webpack', () => {
   })
 })
 
-it('uses custom webpack', () => {
-  return run([], { cwd: fixture('config') }).then(result => {
-    expect(result.out).toContain('Package size: 2.2 KB')
-    expect(result.code).toEqual(0)
-  })
-})
-
```

Comment:
This tests was removed as it was a duplicate of the below test

Reply:
We need both tests since one test API and second test CLI (integration test).

Reply:
Good point, I will put it back.